part-4/model-building-components.md

```diff
@@ -418,63 +418,104 @@ Models generated in **PK-Sim**® make extensive **use of tags**: For example, op
+{% endhint %}
 
-Molecule start values can also be imported from Excel-files which is detailed below.
+- **Extend from Initial Conditions Building Block**: Adds entries from the Initial Conditions BB previously exported to pkml. New values are always *added*, existing values are *overwritten*.
+- **Extend from Expression Profile Building Block**: Adds entries from an Expression Profile BB previously exported to pkml. New values are always *added*, existing values are *overwritten*. Useful when creating a custom expression profile as initial condition.
 
-To edit a molecule start value building block, double-click on it or use the context menu in the Building Block Explorer and select **Edit** <img src="../assets/icons/Edit.svg" data-size="line">. An edit window opens, analogue to the one used when creating new start values. You can now
+The editor of the IC BB has the following buttons (multi-select of the rows is possible):
 
-* manually override the start concentrations or dimensions for every molecule in every container;
-* manually change the "IsPresent" state for each molecule;
-* use the <img src="../assets/icons/ExtendParameterStartValues.svg" data-size="line"> **Extend** ribbon button to automatically add new molecules in case more of them have been created or loaded in the molecules building block after initially creating the start values or executing the last **Extend** command;
-* use the combobox "Is Present" in the upper right to make global selections for the "IsPresent" state for all molecules, where you can check or uncheck it for all molecules or selected molecules, giving you a high degree of flexibility in defining the presence of molecules only in desired organs. The <img src="../assets/icons/OK.svg" data-size="line"> **Apply** button has to be pressed after a global selection. Manual changes (see second bullet) are still possible afterwards.
+- **Delete**: Removes the selected entries
+- **Refresh values**: 
+- **Present**: Sets the IsPresent status of all selected entries to `true`
+- **Not Present**: Sets the IsPresent status of all selected entries to `false`.
 
 {% hint style="info" %}
-Restricting the presence of molecules to certain organs may improve your computing performance, but use it carefully to keep your model valid!
+A molecule is considered as **not present** in a container in a simulation if the simulation configuration contains no IC BB where the molecule is marked as present in the container.
 {% endhint %}
 
+{% hint style="info" %}
 Restricting the presence of molecules to certain organs may improve your computing performance, but use it carefully to keep your model valid!
+{% endhint %}
+
+- **Negative Values Allowed/Not Allowed**: Toggles the setting if negative values are allowed for the selected entries. If disabled (by default for all molecules), a simulation will fail if the species will become negative during the simulation.
```

Comment:
would replace _species_ with _molecule amount_:
_...simulation will fail if the molecule amount becomes negative during the simulation._